src/sentry/static/sentry/app/views/performance/transactionSummary/userStats.tsx

```diff
@@ -84,6 +84,9 @@ function UserStats({totals, location, organization, transactionName}: Props) {
       <div>
         <SectionHeading>{t('Apdex Score')}</SectionHeading>
         <StatNumber>{apdex}</StatNumber>
+        <Link to="">
```

Comment:
@Zylphrex How do I link this to performance settings and reflect the stored value?